### DISCUSSION TITLE
Sending emails without requiring a pay for service?

### DISCUSSION THREAD

**Discussion by PerkinsWork** [microrealestate/microrealestate#92] [Ideas]
Would it be possible to get a feature where I can just use my Gmail account, or maybe just spoof my email address so there's no need for a mailing service?

**Comment by camelaissani** [comment]
Functionality added 😉

**Reply by ridhoswasta** [reply]
hello! is there docs how to use custom SMTP server? 
Thanks!

**Reply by camelaissani** [reply]
Hello currently it is not possible to use a custom SMTP server. Only Gmail and Mailgun can be used.

**Reply by camelaissani** [reply]
Now a custom SMTP server can be used to send emails. Thanks to @zeronounours for his contribution to the project.

**Comment by timastana** [comment]
Hello, It would be great to have a notification system that displays a message when there is an error or a successful execution. For example, when an email hasn't been sent. Maybe using a plugin like Notify.js or toastr for this purpose, which can be used universally across the application.
